### PR TITLE
chore(release): pull hotfix-release/v1.54.4 into main

### PR DESCRIPTION
**Pull request by devops-github-rudderstack**
:crown: *An automated PR*

### CODE REVIEW

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3045/test-report.html

**Review by shrouti1507 — DISMISSED** [review]

**Review by yashasvibajpai — APPROVED** [review]

File: test/integrations/destinations/reddit/processor/data.ts [hunk 1]
```diff
@@ -126,7 +126,10 @@ export const data = [
                         screen_dimensions: {},
                       },
                       event_metadata: {
-                        item_count: 3,
+                        item_count: 2,
+                        currency: 'USD',
+                        value: 15,
+                        value_decimal: 0.15,
```
Comment (krishna2020) [inline test/integrations/destinations/reddit/processor/data.ts:132]: what's expected out of `value_decimal` ? how 0.15 is interpreted, can we re-check docs ?

Comment (yashasvibajpai) [reply test/integrations/destinations/reddit/processor/data.ts:132]: Earlier this block was being omitted entirely  due to incorrect implementation, hence it is coming in tests

Comment (krishna2020) [reply test/integrations/destinations/reddit/processor/data.ts:132]: From the docs shared by @yashasvibajpai , `value_decimal` is not just dividing by 100, it's for minor to major currency conversion

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3045/test-report.html

**Review by yashasvibajpai — APPROVED** [review]

**Review by krishna2020 — APPROVED** [review]

**Pull request merged**